pkg/drivers/equinix-metal/equinix-metal.go

```diff
@@ -66,99 +66,99 @@ func NewDriver(hostName, storePath string) *Driver {
 func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 	return []mcnflag.Flag{
 		mcnflag.StringFlag{
-			Name:   "packet-api-key",
-			Usage:  "Packet api key",
-			EnvVar: "PACKET_API_KEY",
+			Name:   "metal-api-key",
+			Usage:  "Equinix Metal api key",
+			EnvVar: "METAL_API_KEY",
```

Comment:
While we are renaming things, maybe we should take this opportunity to rename this value to `METAL_AUTH_TOKEN` (consistent Terraform and packngo (if we assume a rename of PACKET_AUTH_TOKEN there)).